sigmastate/src/main/scala/sigmastate/lang/exceptions/SigmaBuilderExceptions.scala

```diff
@@ -5,6 +5,3 @@ import sigmastate.lang.SourceContext
 final class ConstraintFailed(message: String, source: Option[SourceContext] = None)
   extends BuilderException(message, source)
 
```

Comment:
Rename the file to "ConstraintFailed.scala"

Reply:
Done